### PR TITLE
Resample portfolio data to consistent time-interval before charting

### PR DESCRIPTION
**Pull request by lyricnz**
Continue from #165 (which attempts to solve #157)

This patch resamples all the data from the exchange before rendering it, so it has a consistent time-interval. This removes problems with dissimilar time-ranges for data.

### CODE REVIEW

**Review by miguelmota — APPROVED** [review]

**Comment by miguelmota** [comment]
This closes https://github.com/miguelmota/cointop/pull/159 as well, correct?

File: cointop/chart.go [hunk 1]
```diff
@@ -2,16 +2,24 @@ package cointop
...
 	"time"
 
 	"github.com/miguelmota/cointop/pkg/chartplot"
+	"github.com/miguelmota/cointop/pkg/timedata"
 	"github.com/miguelmota/cointop/pkg/timeutil"
 	"github.com/miguelmota/cointop/pkg/ui"
 	log "github.com/sirupsen/logrus"
 )
 
+// Time-series data for a Coin used when building a Portfolio view for chart
```
Comment (miguelmota) [inline cointop/chart.go:17]: From [Effective Go](https://golang.org/doc/effective_go)

>  Doc comments work best as complete sentences, which allow a wide variety of automated presentations. The first sentence should be a one-sentence summary that starts with the name being declared. 

suggestion:

```go
// PriceData is the time-series data for a Coin used when building a Portfolio view for chart
```

Comment (lyricnz) [reply cointop/chart.go]: Thanks!

**Comment by lyricnz** [comment]
> This closes #159 as well, correct?

Yes, this is more complete solution than 159.

File: cointop/chart.go (hunk 1 above)
Comment (miguelmota) [reply cointop/chart.go]: I think [timedata.go](https://github.com/miguelmota/cointop/pull/180/files#diff-70eb352e3f06f42044d76aa4b634d10631ac5d23b5e028d81761d79096eb58d4R13) might also need comment fixes

**Comment by miguelmota** [comment]
Go ahead and merge pull request if you have the ability now @lyricnz

**Pull request merged**